src/preload.js

```diff
@@ -7,17 +7,17 @@ import { renderPaperSelectOptions } from './utils/renderUtils.js';
 window.addEventListener('DOMContentLoaded', () => {
     // render dynamic content
     renderPaperSelectOptions();
-    loadForm();
+    const configuration = loadForm();
 
     // grab DOM elements
     const generate = document.getElementById('generate');
     const preview = document.getElementById('preview');
     const bookbinderForm = document.getElementById('bookbinder');
     const fileInput = document.getElementById('input_file');
-    const inputs = document.querySelectorAll('input, select');
+    const inputs = document.querySelectorAll('input, select, radio, checkbox');
```

Comment:
The changes here won't actually do anything - `radio` and `checkbox` aren't html tag types, they're all under `input`.

Reply:
You are absolutely correct ~ removed in [bbae689](https://github.com/momijizukamori/bookbinder-js/pull/63/commits/bbae689669773456aca53f4e4f01e177efecc03a)